src/renderer/contexts/SchemaProvider.tsx

```diff
@@ -6,11 +6,15 @@ const SchemaContext = React.createContext<{
   schema?: DatabaseSchemas;
   currentDatabase?: string;
   setCurrentDatabase: (v: string) => void;
+  reloadSchema: () => void;
 }>({
   schema: {},
   setCurrentDatabase: () => {
     throw 'Not implemented';
   },
+  reloadSchema: () => {
+    throw 'Not implemented';
+  },
```

Comment:
https://github.com/invisal/query-master/blob/main/src/libs/NotImplementCallback.ts

We can use

```
setCurrentDatabase: NotImplementCallback,
reloadSchema: NotImlementCallback
```